ext/doc/Cassandra/Timeuuid.php

```diff
@@ -21,55 +21,50 @@
 /**
  * A PHP representation of the CQL `timeuuid` datatype
  */
-final class Timeuuid implements Value, UuidInterface
-{
+final class Timeuuid implements Value, UuidInterface {
+
     /**
      * Creates a timeuuid from a given timestamp or current time.
-     *
      * @param int $timestamp Unix timestamp
      */
-    public function __construct($timestamp = null) {}
+    public function __construct($timestamp) { }
 
     /**
-     * The type of this timeuuid.
-     *
-     * @return Type
+     * Returns this timeuuid as string.
+     * @return string timeuuid
      */
-    public function type() {}
+    public function __toString() { }
 
     /**
-     * Returns this timeuuid as string.
-     *
-     * @return string timeuuid
+     * The type of this timeuuid.
+     * @return Type
      */
-    public function __toString() {}
+    public function type() { }
 
     /**
      * Returns this timeuuid as string.
-     *
      * @return string timeuuid
      */
-    public function uuid() {}
+    public function uuid() { }
 
     /**
      * Returns the version of this timeuuid.
-     *
      * @return int version of this timeuuid
      */
-    public function version() {}
+    public function version() { }
 
     /**
      * Unix timestamp.
      *
-     * @return int seconds
      * @see time
+     * @return int seconds
```

Comment:
I think @return should come before @see. See the template I wrote up in one of the very early comments.